fe/src/main/java/org/apache/doris/catalog/OlapTable.java

```diff
@@ -148,6 +150,8 @@ public OlapTable() {
         this.indexes = null;
       
         this.tableProperty = null;
+
+        this.replicationNum = null;
```

Comment:
I think you can add this `this.replicationNum` to the `tableProperty` so that we do not need to update the FeMetaVersion.

Reply:
The current analyzer will remove all the table property after analyzed, include replication_num, so is it a better way to change this design?

Reply:
ok. I will fix it